<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoPlayer</class>
 <widget class="QWidget" name="VideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Player de VÃ­deo</string>
  </property>
  <widget class="Phonon::VideoPlayer" name="video_player">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>671</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="Phonon::SeekSlider" name="seek_slider">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_play">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/button_blue_play.png</normaloff>:/i/button_blue_play.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="Phonon::VolumeSlider" name="volume_slider">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>390</y>
     <width>114</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/button_blue_stop.png</normaloff>:/i/button_blue_stop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pause">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/button_blue_pause.png</normaloff>:/i/button_blue_pause.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <zorder>btn_pause</zorder>
  <zorder>video_player</zorder>
  <zorder>seek_slider</zorder>
  <zorder>btn_play</zorder>
  <zorder>volume_slider</zorder>
  <zorder>btn_stop</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
